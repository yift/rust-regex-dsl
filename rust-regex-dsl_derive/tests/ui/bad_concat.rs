pub use rust_regex_dsl::regex_dsl;

fn main() {
    let regex = regex_dsl! {
        concat,
    };
    let regex = regex_dsl! {
        concat(),
    };
    let regex = regex_dsl! {
        concat("one"),
    };
    let regex = regex_dsl! {
        concat(regex("["), "two"),
    };
    println!("{}", regex);
}