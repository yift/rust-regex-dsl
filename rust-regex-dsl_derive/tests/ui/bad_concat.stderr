error: Unknown word: concat
 --> tests/ui/bad_concat.rs:5:9
  |
5 |         concat,
  |         ^^^^^^

error: unexpected end of input, Concat must have at least two elements
 --> tests/ui/bad_concat.rs:8:16
  |
8 |         concat(),
  |                ^

error: unexpected end of input, Concat must have at least two elements
  --> tests/ui/bad_concat.rs:11:21
   |
11 |         concat("one"),
   |                     ^

error: regex parse error:
           [
           ^
       error: unclosed character class
  --> tests/ui/bad_concat.rs:14:22
   |
14 |         concat(regex("["), "two"),
   |                      ^^^
